### ISSUE TITLE
Swift support

### ISSUE THREAD

**Issue by tonyxiao** [gangverk/GVUserDefaults#21]
Objective c properties of course do not quite work in swift, but can similar idea be accomplished in swift? So that it is easy and convenient to access NSUserDefaults via properties in swift?

**Comment by kevinrenskers** [comment]
Good idea. Sadly I haven't touched Swift yet, and I don't think we won't in any of our production apps for some time. So Swift support would have to be added by a contributor, but maybe it would make more sense to make a brand new project specifically for Swift?
